src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.handlers.BalanceHandler;
+import core.basesyntax.strategy.handlers.OperationHandler;
+import core.basesyntax.strategy.handlers.PurchaseHandler;
+import core.basesyntax.strategy.handlers.ReturnHandler;
+import core.basesyntax.strategy.handlers.SupplyHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private final Map<Operation, OperationHandler> map = new HashMap<>();
+
+    {
+        map.put(Operation.BALANCE, new BalanceHandler());
+        map.put(Operation.SUPPLY, new SupplyHandler());
+        map.put(Operation.PURCHASE, new PurchaseHandler());
+        map.put(Operation.RETURN, new ReturnHandler());
+    }
```

Comment:
Create a contractor that will have one parameter Map<Operation, OperationHandler> and initialize your map via that constructor. Initialize block is a bad practice